rerun_cpp/src/rerun/blueprint/components/column_share.cpp

```diff
@@ -15,6 +15,26 @@ namespace rerun {
         return datatype;
     }
 
+    Result<std::shared_ptr<arrow::Array>> Loggable<blueprint::components::ColumnShare>::to_arrow(
```

Comment:
did you change the order of the functions just to get a bigger diff ;)

Reply:
lol yeah I thought about avoiding that but it just happened because the last function is now optionally appended to the vec of methods on `Loggable`. It's annoying but 🤷